bin/modify_existing_sample.py

```diff
@@ -48,22 +47,10 @@ def main():
 
     # Load the config_file from default location
     load_config()
-    sample_submitter = SampleMetadataSubmitter(args.metadata_file, submit_type=(args.action,))
+    sample_submitter = SampleJSONSubmitter(args.metadata_file, submit_type=(args.action,))
     sample_name_to_accession = sample_submitter.submit_to_bioSamples()
-    if args.action == 'derive':
```

Comment:
Is this not needed for the JSON metadata?  (Sorry if this is a dumb question, tbh I can't really remember what we use `derive` for....)